packages/frontend/scss/message/_message.scss

```diff
@@ -409,6 +411,13 @@
     &.outgoing {
       margin-left: auto;
     }
+    /**
+     * on small screens the limiting factor is the
+     * width not the height (except for stickers)
+     */
+    img.portrait:not(.sticker) {
```

Comment:
Again, `.message:not(.type-sticker)` can be used here it seems. But alright.

Reply:
.type-sticker is a class on the higher order message item, not on the image